### PR TITLE
Onboarding updates

### PR DESCRIPTION
**Pull request by ctro**
- Added links to authentication and identity proofing intro videos
- Added more Slack groups and onboarding reminders

### CODE REVIEW

File: _articles/slack.md [hunk 1]
```diff
@@ -28,7 +28,10 @@ These handles ping oncall engineers, use for emergencies or urgent items.
 - `@login-partnerships-team` Group to address login.gov partnerships team members
 - `@login-product-team` Group to address login.gov product managers
 - `@login-ux-team` Group for the design/UX team
+- `@login-appdev-team` Group for application development teams
```
Comment (zachmargolis) [inline _articles/slack.md:31]: should we remove `@login-eng` since that is also the "app team" ?

**Review by zachmargolis — APPROVED** [review]
LGTM, one quick note, not a blocker

**Review by orenyk — APPROVED** [review]
Agree with @zachmargolis, looks good (and I made sure that the video links were correct). :shipit:

File: _articles/slack.md (hunk 1 above)
Comment (ctro) [reply _articles/slack.md:31]: `login-eng` looks like just engineers.  There was request for an `appdev-team` group that includes all members of Ada, Grace, and Katherine. (UX, Product, etc is included).    How can I make that more clear?

Comment (zachmargolis) [reply _articles/slack.md:31]: Oh I have no idea, maybe let's leave in both for the time being then

**Pull request merged**